apps/daffio/src/app/content/homepage/component/homepage.component.scss

```diff
@@ -1,11 +1,12 @@
-@import 'daff-util';
+@import 'theme';
```

Comment:
remove?

Reply:
Resolved in bee3c33bacc52d9bda5982546821edf17ab62b6d